### PR TITLE
Project details functionality

### PR DESCRIPTION
**Pull request by Willian-Berrocal**
> See project button functionality added

> Details modal window created for both mobile and desktop

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1,19 +1,148 @@
...
+
+detailsButton4.addEventListener('click', showDetails);
+detailsButton4.numpro = 3;
+
+// Hide details
+
+const closeDetails = document.querySelector('.close-details');
+
+function hideDetails() {
+  document.querySelector('.app-details').classList.replace('section-show', 'section-hide');
```
Comment (CollinsTatang) [inline main.js:145]: - [ ] Please implement the close functionality on the mobile pop-up screen. The user can't close the pop-up on a mobile device
As shown in the [figmal](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)
mobilescre

File: index.html [hunk 2]
```diff
@@ -11,7 +11,7 @@
   <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">  
 </head>
 <body>
-  <aside class="app-menu container-v menu-hide">
+  <aside class="app-menu container-v section-hide">
```
Comment (CollinsTatang) [inline index.html:14]: - [ ] Please display your desktop navbar as shown below on the [figmal design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)
nav

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @Willian-Berrocal, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required changes 
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips that can improve your code quality and help you get a better job.
_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.js (hunk 1 above)
Comment (Willian-Berrocal) [reply main.js:145]: Hi, I don't understand what you are asking for, my closing button does close the pop-up window, please check it again by refreshing your browser.

File: index.html (hunk 2 above)
Comment (Willian-Berrocal) [reply index.html:14]: Hi, my navbar in the desktop version looks like that, please check it again. And the code you are referring to is not my navbar, that's my menu that opens when clicking on the burger button.

File: main.js [hunk 3]
```diff
@@ -1,19 +1,148 @@
...
 }
 
-burgerMenu.addEventListener('click', showMenu);
\ No newline at end of file
+closeDetails.addEventListener('click', hideDetails);
```
Comment (omarsalem7) [inline main.js]: - [ ] As stated by the previous reviewer, Kindly fix your design in mobile version @375px to the problem which is the close icon is hidden, so I can't close the popup.
> Your design
image

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @Willian-Berrocal , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

Almost near

 ## Highlights
✔️ All linter checks pass
✔️ Descriptive pull request

 ## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.js (hunk 3 above)
Comment (Willian-Berrocal) [reply main.js]: Hi, thank you for your kind response, I tried to replicate the error in Chrome with the dimensions you are mentioning but I still cannot see it, I've tried also with Opera, Firefox, and Edge, so I'm really confused about it, but still I've made some changes so the card should stick to the top in any case and you can see the button.

Screenshot 2022-10-12 201421

**Review by FrederickMih — APPROVED** [review]
# Approved :heavy_check_mark: 

Hi @Willian-Berrocal , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :green_circle: 

- Great job for making sure that the project meets all requirements :+1: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.js (hunk 1 above)
Comment (CollinsTatang) [reply main.js:145]: Hi @Willian-Berrocal 
I think it now work fine after the changes
Thanks

**Pull request merged**